evaluate/task/test-integration/task.go

```diff
@@ -0,0 +1,61 @@
+package testintegration
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/symflower/eval-dev-quality/evaluate/metrics"
+	evaluatetask "github.com/symflower/eval-dev-quality/evaluate/task"
+	tasktesting "github.com/symflower/eval-dev-quality/evaluate/task/testing"
+	"github.com/symflower/eval-dev-quality/language/golang"
+	"github.com/symflower/eval-dev-quality/log"
+	"github.com/symflower/eval-dev-quality/model/symflower"
+	"github.com/symflower/eval-dev-quality/tools"
+	toolstesting "github.com/symflower/eval-dev-quality/tools/testing"
+)
+
+func TestTaskWriteTestsRun(t *testing.T) {
+	toolstesting.RequiresTool(t, tools.NewSymflower())
+
+	validate := func(t *testing.T, tc *tasktesting.TestCaseTaskWriteTests) {
+		t.Run(tc.Name, func(t *testing.T) {
+			resultPath := t.TempDir()
+
+			_, logger := log.Buffer()
+			repository, cleanup, err := evaluatetask.TemporaryRepository(logger, tc.TestDataPath, tc.RepositoryPath)
+			assert.NoError(t, err)
+			defer cleanup()
+
+			taskWriteTests := evaluatetask.NewTaskWriteTests(logger, resultPath, tc.Model, tc.Language)
+			tc.Validate(t, taskWriteTests, repository, resultPath)
+		})
```

Comment:
Would move most of this initialization into the validation of `TestCaseTaskWriteTests`... cause it will be the same for other tasks... i.e. setting up a repo and initializing the task will happen i.e. for the code repair as well.

Reply:
Not possible as this would yield a cyclic import again. `evaluate/task` imports `evaluate/task/testing` imports `evaluate/task`.

There is another refactoring needed at some point. But it's good enough for now IMO.

Reply:
oh ffs...

Reply:
Wait but where is `evaluate/task` importing `evaluate/task/testing/? You are in another package `evaluate/task/test-integration` here :thinking: